buildalias/thundernetes-buildalias.toml

```diff
@@ -0,0 +1,17 @@
+[[alias]]
+
+[[alias.sampleAlias]]

```

Comment:
not sure I'm a fan of the `sampleAlias` naming. This looks  more than an alias entry, should we call it something like that? Thanks!